python/pyspark/sql/connect/README.md

```diff
@@ -0,0 +1,34 @@
+
+# [EXPERIMENTAL] Spark Connect
+
+**Spark Connect is a strictly experimental feature and under heavy development.
+All APIs should be considered volatile and should not be used in production.**
+
+This module contains the implementation of Spark Connect which is a logical plan
+facade for the implementation in Spark. Spark Connect is directly integrated into the build
+of Spark. To enable it, you only need to activate the driver plugin for Spark Connect.
+
+
+
+
+## Build
+
+1. Build Spark as usual per the documentation.
+2. Build and package the Spark Connect package
+   ```commandline
```

Comment:
Seems like we should switch `commandline` to `bash` or `sh` (https://www.rubycoloredglasses.com/2013/04/languages-supported-by-github-flavored-markdown/)

Reply:
Done.